pages/index.js

```diff
@@ -50,62 +56,159 @@ export const getServerSideProps = async () => {
   };
 };
 
-function HomePage({
+const getStateData = async (state) => {
+  const shootingsByCounty = await universalAPIFetch(
+    `count/shootings/statecounty/${state}`
+  );
+  const shootingsOverTime = await universalAPIFetch(
+    `count/shootings/overtime/${state}`
+  );
+  const last20Items = await universalAPIFetch(`shootings/last20/${state}`);
+  const topPoliceDepartments = await universalAPIFetch(
+    `count/shootings/pd/${state}`
+  );
+
+  return {
+    shootingsByCounty,
+    shootingsOverTime,
+    topPoliceDepartments,
+    last20Items,
+  };
+};
+
+export const getServerSideProps = async () => {
+  const nationalData = await getNationalData();
+  return nationalData;
+};
+
+function Dashboard({
   shootingsByState,
-  shootingsPerCapita,
+  shootingsByGeo,
   shootingsOverTime,
   topPoliceDepartments,
   last20Items,
+  selectedState,
+  setSelectedState,
+  setIsPerCapita,
 }) {
-  const [isPerCapita, setIsPerCapita] = useState(false);
-
   return (
     <Container>
       <Head />
+      <Row>
+        <Col sm="auto">
+          <h2>Police Killings in 2020</h2>
+        </Col>
+        <Col sm="auto" style={filterColumnStyle}>
+          {selectedState ? (
+            <h4>
+              <Badge pill variant="info" style={{ verticalAlign: "bottom" }}>
+                {selectedState}{" "}
+                <CloseIcon
+                  fontSize="inherit"
+                  onClick={() => setSelectedState(null)}
+                />
+              </Badge>
+            </h4>
+          ) : (
+            <Form.Check
+              type="switch"
+              id="custom-switch"
+              label="Per Capita"
+              onChange={(val) => {
+                setIsPerCapita(val.target.checked);
+              }}
+            />
+          )}
+        </Col>
+      </Row>
       <Row style={rowStyle}>
         <Col lg={8}>
-          <h2>Police Killings by State</h2>
-          <Form.Check
-            type="switch"
-            id="custom-switch"
-            label="Per Capita"
-            onChange={(val) => {
-              setIsPerCapita(val.target.checked);
-            }}
-          />
+          <h3>Police Killings by State</h3>
+          <Row>
+            <Col></Col>
+          </Row>
           <BrutalityMap
-            data={isPerCapita ? shootingsPerCapita : shootingsByState}
+            data={shootingsByState}
+            selectState={setSelectedState}
+            selectedState={selectedState}
           />
         </Col>
         <Col lg={4}>
-          <h2>Recent Police Killings</h2>
+          <h3>Recent Police Killings</h3>
```

Comment:
If you do end up nudging the first `h2` up to an `h1`m these can be knocked back to `h2` elements.